apps/dashboard/src/components/conditions-editor/variable-select.tsx

```diff
@@ -54,7 +55,7 @@ const VariablesList = React.forwardRef<HTMLUListElement, VariablesListProps>(
               }}
             >
               <Code2 className="text-feature size-3 min-w-3" />
-              {option.label}
+              <TruncatedText>{option.label}</TruncatedText>
```

Comment:
Fix for the long variable names, the truncated text will be shown in tip on hover